examples/basic-tracer-node/README.md

```diff
@@ -1,7 +1,7 @@
 # Overview
 
 This example shows how to use [@opentelemetry/tracing](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing) to instrument a simple Node.js application - e.g. a batch job.
-It supports exporting spans either to [Zipkin](https://zipkin.io) or to [Jaeger](https://www.jaegertracing.io).
+It supports exporting spans either to `Console` and to [Jaeger](https://www.jaegertracing.io).
```

Comment:
```suggestion
It supports exporting spans to `Console` and [Jaeger](https://www.jaegertracing.io).
```

Reply:
I would also mention that jaeger needs to be running on the localhost and default port.